### ISSUE TITLE
Some questions about reproducing ours-sync

### ISSUE THREAD

**Issue by Liy-s** [AniSundar18/AlignedForensics#10]
Hello! Thank you for your excellent work and open source code!
I am trying to reproduce "ours-sync" training with your code and dataset released on your huggingface page <https://huggingface.co/datasets/AniSundar18/aligned_forensic_trainingdata>. My training scripts is:
`CUDA_VISIBLE_DEVICES=6 python train.py --name=$LDM_DS_NAME --arch res50nodown --cropSize 96  --norm_type resnet --resize_size 256 --resize_ratio 0.75 --blur_sig 0.0,3.0 --cmp_method cv2,pil --cmp_qual 30,100 --resize_prob 0.2 --jitter_prob  0.8 --colordist_prob 0.2 --cutout_prob 0.2 --noise_prob 0.2 --blur_prob 0.5 --cmp_prob 0.5 --rot90_prob 1.0 --dataroot $DATASET --batch_size 32 --earlystop_epoch 10 --use_inversions --seed 17 --batched_syncing` .
During the validation before training starts, my val acc was only **12.63%**, and I’m not sure whether that’s a normal value. However, even after training for a few epochs, the val acc still didn’t go above **20%**, which makes me worry that something might be going wrong in my reproducing process.

I tried both single-GPU training and also modified the code to support parallel training, but I ran into the same issue in both cases. Could you please help me understand what might be happening? Thank you very much!

**Comment by AniSundar18** [comment]
> Hello! Thank you for your excellent work and open source code! I am trying to reproduce "ours-sync" training with your code and dataset released on your huggingface page https://huggingface.co/datasets/AniSundar18/aligned_forensic_trainingdata. My training scripts is: `CUDA_VISIBLE_DEVICES=6 python train.py --name=$LDM_DS_NAME --arch res50nodown --cropSize 96 --norm_type resnet --resize_size 256 --resize_ratio 0.75 --blur_sig 0.0,3.0 --cmp_method cv2,pil --cmp_qual 30,100 --resize_prob 0.2 --jitter_prob 0.8 --colordist_prob 0.2 --cutout_prob 0.2 --noise_prob 0.2 --blur_prob 0.5 --cmp_prob 0.5 --rot90_prob 1.0 --dataroot $DATASET --batch_size 32 --earlystop_epoch 10 --use_inversions --seed 17 --batched_syncing` . During the validation before training starts, my val acc was only **12.63%**, and I’m not sure whether that’s a normal value. However, even after training for a few epochs, the val acc still didn’t go above **20%**, which makes me worry that something might be going wrong in my reproducing process.
> 
> I tried both single-GPU training and also modified the code to support parallel training, but I ran into the same issue in both cases. Could you please help me understand what might be happening? Thank you very much!


Hi, Thank you for your interest,

Yes, that does seem like something might have gone wrong, Could you help me out by sharing the following statistics, I will be able to provide a better response after that.

- Number of training real and fake images, number of validation real/fake images. (training should be like 170k each)
- How is the training loss behaving? i.e how many epochs did you train on? Sharing the loss curve could also help with diagnosis.
- Just let me know any other components which were potentially changed.

Also my experiments use a batch size of 64 real and 64 reconstructions. Although I am not sure if that alone is causing this.
